tests/unit_tests/test_stats.py

```diff
@@ -85,6 +113,14 @@ def test_maxwell():
     assert d.theta == theta
     assert len(d) == 1
 
+    exp_mean = 3/2 * theta
+
+    # sample maxwell distribution
+    n_samples = 1_000_000
+    samples = d.sample(n_samples, seed=100)
+    assert samples.mean() == pytest.approx(exp_mean, rel=1e-02)
+
+
```

Comment:
Just wondering if it might be worth adding a test to check that that change to the seed results in a changed sample?

```suggestion
    # A second sample with a different seed
    samples_2 = d.sample(n_samples, seed=200)
    assert samples_2.mean() == pytest.approx(exp_mean, rel=1e-02)
    assert samples_2.mean() != samples.mean()


```